docs/index.rst

```diff
@@ -161,21 +155,31 @@ Contents
 
 .. toctree::
    :maxdepth: 2
-   :caption: Additional Material
+   :caption: Avisory content
 
+   security-considerations.rst
+   bugs.rst
    050-breaking-changes.rst
    060-breaking-changes.rst
    070-breaking-changes.rst
    080-breaking-changes.rst
```

Comment:
Says in the description that the list of breaking changes should be one link only. What did you mean by that?

Reply:
I assume this will then come in a later PR?

Reply:
This is a draft PR aka WIP.

Reply:
Under the `Entries` section in the doc you linked, it seems like you can manually generate the names and targets of the drop down, but I think your second suggestions (1 page with subheaders by version) would probably be easier to maintain, and the toc tree itself would then be automatically generated.

Reply:
It would be a really long page (reverse sorted - from 0.8 to oldest), but I like the idea of being auto-generated.
We can also auto-generate the menu with a regular expression according to docs.

Reply:
@nikola-matic There: https://github.com/ethereum/solidity/pull/14091/files take a look.